allcontainers.js

```diff
@@ -151,16 +151,22 @@ function toRegExp(obj) {
   return new RegExp(obj)
 }
 
+function ifRegex(value) {
+  var result = /^\/(.*)\/$/.exec(value);
+  return  result ? new RegExp(result[1]) : value;
+}
```

Comment:
Maybe this needs to get into https://github.com/mcollina/docker-allcontainers/blob/master/allcontainers.js#L142-L152?